docs/release-notes/8.11.asciidoc

```diff
@@ -0,0 +1,61 @@
+[[release-notes-header-8.11.0]]
+== 8.11
+
+[discrete]
+[[release-notes-8.11.0]]
+=== 8.11.0
+
+[discrete]
+[[known-issue-8.11.0]]
+==== Known issues
+* Duplicate rules are displayed in MITRE ATT&CK® technique cells ({issue}167929[#167929]).
+* The incorrect rule count is shown in MITRE ATT&CK® technique cells ({issue}167930[#167930]).
+* The incorrect MITRE ATT&CK® sub-technique is applied after saving a rule ({issue}170347[#170347]).
```

Comment:
Consider consolidating these into succinct bullets a little:
```suggestion
For MITRE ATT&CK® technique cells:
* Duplicate rules are displayed ({issue}167929[#167929]).
* The incorrect rule count is shown ({issue}167930[#167930]).
* The incorrect MITRE ATT&CK® sub-technique is applied after saving a rule ({issue}170347[#170347]).
```

Reply:
I think the last item is tied to the rule creation flow, but the first two can be grouped into a list. 

```suggestion
* For MITRE ATT&CK® technique cells:
** Duplicate rules are displayed ({issue}167929[#167929]).
** The incorrect rule count is shown ({issue}167930[#167930]).
* The incorrect MITRE ATT&CK® sub-technique is applied after saving a rule ({issue}170347[#170347]).
```